server/routes/users.js

```diff
@@ -25,6 +29,32 @@ const signup = async (request, reply) => {
    }
 }
 
+const login = async (request, reply) => {
+   try {
+      const userMail = request.body.mail;
+      const userPassword = request.body.userPassword;
+      const candidateUser = await checkUserExist(userMail);
+      if (candidateUser === undefined) {
+         reply.status(404).send({
+            'err': `There is no user ${userMail}`
```

Comment:
As far as I understand, we're sticking with `message` property in responses (see `signup` function above):

```suggestion
            message: `There is no user ${userMail}`
```